lib/OpenQA/Schema/Profiler.pm

```diff
@@ -32,6 +32,13 @@ sub query_end {
     log_debug(sprintf("[DBIC] Took %.8f seconds: %s", $elapsed, $sql));
 }
 
+sub print {
```

Comment:
I wonder, you put a description in the github pull request but not in the commit message. For this we should keep in mind that git commit messages should be seen as more persistent than the github PR description. Even more I think that you could put it into here to describe the method. WDYT?

Reply:
I would appreciate a comment explaining the motivation as well, to make it clear to the next person wondering why it exists and if it isn't redundant or dead code.

Reply:
Added a comment to the code and a link to the issue to the commit message